src/components/file-upload/file-upload.ts

```diff
@@ -291,6 +296,10 @@ export class MFileUpload extends ModulVue {
         return this.rejectedFiles.length !== 0;
     }
 
+    private get hasAllowedExtensions(): boolean {
+        return this.allowedExtensions !== undefined && !!this.allowedExtensions.length;
```

Comment:
Avec la valeur par défaut, ut pourrais juste faire return this.allowedExtensions.length > 0